roles/edpm_download_cache/tasks/container_images.yml

```diff
@@ -1,21 +1,5 @@
 ---
 
-- name: Install podman
-  ansible.builtin.include_role:
-    name: osp.edpm.edpm_podman
-    tasks_from: install.yml
-  tags:
-    - edpm_podman
-    - download_cache
-
-- name: Login to container registries if needed
-  ansible.builtin.include_role:
-    name: osp.edpm.edpm_podman
-    tasks_from: login.yml
-  tags:
-    - edpm_podman
-    - download_cache
```

Comment:
I feel like we may still want the login task here. Would the login credentials ever expire?

Reply:
I hadn't tought about it, but it could be a problem, since we are going to perform the installation of the rpms between the login task and the download of the containers. Let me move this task back in this file.